examples/kubernetes/connectivity-check/echo-servers.cue

```diff
@@ -15,15 +15,15 @@ _echoDeploymentWithHostPort: _echoDeployment & {
 
 // Regular service exposed via ClusterIP.
 deployment: "echo-a": _echoDeployment & {
-	_serverPort: "8080"
+	_serverPort:      "8080"
```

Comment:
unrelated to this PR, this is due to `cue fmt`, which we should run in CI as well.